app/javascript/packages/form-steps/form-error.ts

```diff
@@ -9,18 +11,23 @@ export interface FormErrorOptions {
    * Field associated with the error.
    */
   field?: string;
+
+  messageProcessor?: (message: string) => string | ReactNode;
 }
 
 class FormError extends Error {
   field?: string;
 
   isDetail: boolean;
 
+  messageProcessor?: (message: string) => string | ReactNode;
```

Comment:
I don't think we need to include this argument here. We're only using this argument once.

Reply:
@charleyf , in `form-steps`

https://github.com/18F/identity-idp/blob/b0ac05ac3958562304825de59dbe0785c2c1bc89/app/javascript/packages/form-steps/form-steps.tsx#L239-L239

It defined as `Error` and used to render `Alert`, so need to include in `FormError` as part of subclass, cannot include in built-in class `Error`.